src/components/core/Alert.vue

```diff
@@ -56,8 +55,12 @@ export default {
   },
 
   methods: {
+    ...mapActions(['setAlert']),
     getColor (type) {
       return this.colors.get(type) || ''
+    },
+    closeAlert () {
+      this.setAlert(null)
```

Comment:
This is fixing #625 . See the `@click` in the Vuetify Alert component, which is bound to this `closeAlert` function, effectively removing the alert from the store too :+1: e2e test added for this.